css/styles.css

```diff
@@ -60,8 +128,23 @@ thead > tr {
   text-decoration: underline #1e2022;
 }
 
+#total-calculated {
+  margin-top: -1rem;
+  color: #f0f5f9;
+  font-weight: lighter;
+  padding-bottom: 5rem;
+}
+
+#total-worth {
+  padding-right: 15rem;
```

Comment:
You shouldn't really be using anything more than `5rem` for `padding` & `margin`. If you're doing anything more, it usually means you're doing something "hacky" to get this to work.